app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -258,6 +263,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             authorLayout.setVisibility(GONE);
         }
 
+        if(applicationKvStore.getBoolean("login_skipped") == true){
```

Comment:
Please remove `== true`.
Shorter and easier to read :-)

Reply:
Done!